README.md

```diff
@@ -230,41 +230,50 @@ module.exports = {
 	Placeholder for the next version (at the beginning of the line):
 	### **WORK IN PROGRESS**
 -->
+
+### 1.3.1 (2024-12-07)
```

Comment:
Please add this headline as `### **WORK IN PROGRESS**` as described above. The release script automatically does the needed versioning

Reply:
DONE